tests/test-mongoc-collection.c

```diff
@@ -1454,7 +1454,7 @@ test_aggregate_legacy (void *data)
 
    /* no "cursor" argument */
    request = mock_server_receives_command (
-      server, "db", MONGOC_QUERY_NONE,
+      server, "db", MONGOC_QUERY_SLAVE_OK,
```

Comment:
See: https://github.com/mongodb/mongo-c-driver/pull/283#discussion_r41828217 regarding the question about why these aggregation changes were required.